remix/app/components/AppLink/helper.ts

```diff
@@ -0,0 +1,33 @@
+export function setButtonType(color?: string) {
```

Comment:
With this update, shouldn't the param name be changed too?

```suggestion
export function setButtonType(type?: string) {
```

Reply:
Updated here: [9b0ebe5](https://github.com/nimblehq/js-framework-benchmark/pull/130/commits/9b0ebe5ec2af4123c6639502977bb5a584cbc878)